banzai/dbs.py

```diff
@@ -431,6 +431,12 @@ def get_timezone(site, db_address=_DEFAULT_DB):
     return site_list[0].timezone
 
 
+def get_sites(db_address=_DEFAULT_DB):
+    with get_session(db_address=db_address) as db_session:
+        sites = db_session.query(Site.id)
+    return [site[0] for site in sites]
```

Comment:
This should db_session.query(Site.id).all() rather than a list comprehension. Is this needed in banzai or is something this straightforward better in the banzai-web app?

Reply:
I have changed it to all(). I thought it would be better to have calls to the DB contained in Banzai rather than directly calling from banzai-web.